trigger.md

```diff
@@ -0,0 +1,80 @@
+# Finite Triggered Acquisition
+Acquire (`acquire-imaging` [on PyPI](https://pypi.org/project/acquire-imaging/)) is a Python package providing a multi-camera video streaming library focused on performant microscopy, with support for up to two simultaneous, independent, video streams.
+
+This tutorial shows an example of setting up triggered acquisition of a finite number of frames with one of Acquire's supported devices and saving the data to a Zarr file.
+
+## Initialize Acquisitoin 
+
+To start, we'll import `Acquire` and create an acquisition `Runtime` object, which initializes the driver adaptors needed for the supported cameras.
+```
+import acquire
+runtime = acquire.Runtime()
+```
+
+## Configure Camera
+
+All camera settings can be captured by an instance of the `Properties` class, which will be associated will a given camera acquisition. The settings can be stored in a dictionary (e.g: `Properties.dict()`). These settings can be saved to a json files to be subsequently loaded, (e.g. ` Properties(**json.load('acquire.json'))` ). 
+
+The current configuration settings can be checked and assigned to an instance of the `Properties` class with:
+```
+p = runtime.get_configuration() 
+```
+Since `Acquire` supports 2 video streams, each camera, or source, must be configured separately. In this example, we will only use 1 source for the acquisition. The video streams are zero indexed, so to select the device for the first video source use `props.video[0]`. To set the first video stream to the supported Hamamatsu camera, use:
```

Comment:
I don't know how necessary it is to point out that a Python tuple is zero-indexed. I guess we're assuming familiarity with Python?

Reply:
Noted. I updated the text.